src/net.cpp

```diff
@@ -2555,6 +2554,22 @@ void CConnman::GetNodeStats(std::vector<CNodeStats>& vstats) const
     for (CNode* pnode : m_nodes) {
         vstats.emplace_back();
         pnode->CopyStats(vstats.back());
+
+        auto min_ping_time{m_evictionman.GetMinPingTime(pnode->GetId())};
+        if (min_ping_time) {
```

Comment:
Can be shortened to

```suggestion
        if (auto min_ping_time = m_evictionman.GetMinPingTime(pnode->GetId())) {
```

if you think that's better. Same below.

Reply:
Done.